model-optimizer/mo/ops/pooling.py

```diff
@@ -69,7 +96,11 @@ def infer(node: Node):
             node['pad'] = int64_array([[0, 0] for x in range(len(input_shape))])
         if not node.has_valid('pad_spatial_shape'):
             node['pad_spatial_shape'] = node.pad[node.spatial_dims]
-        if not node.has_valid('stride'):
+
+        if is_pool_v2:
+            node['window'] = node.in_port(1).data.get_value()
+            node['stride'] = node.in_port(2).data.get_value()
```

Comment:
A case when some of values is None should be handled.

Reply:
Agree, fixed that.